### PR TITLE
GCP docs: Suggest setting up gke cluster with 3 nodes

### PR DESCRIPTION
**Pull request by vinayvinay**
**What this PR does / why we need it**:

I'm updating docs to get a smooth initial setup. This first example won't run without at least 3 nodes:

https://github.com/SeldonIO/seldon-core-launcher/blob/master/seldon-core/getting_started/deployment/deploy.ipynb

### CODE REVIEW

**Comment by seldondev** [comment]
Hi @vinayvinay. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) or [todo](https://github.com/orgs/todo/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/todo/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository.
</details>

**Review by edshee — APPROVED** [review]
Good spot, thanks for fixing

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[edshee](https://github.com/SeldonIO/seldon-core/pull/3921#pullrequestreview-876374770 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by RafalSkolasinski** [comment]
Not sure if 3 nodes is a strict recommendation, especially here. Could just be a general note that with this flag one can adjust number of nodes.

**Comment by edshee** [comment]
Good point

**Comment by vinayvinay** [comment]
hi @RafalSkolasinski, well, the [Create GKE Cluster](https://cloud.google.com/kubernetes-engine/docs/quickstart) link suggests keeping num-nodes to 1, so if someone like me blindly follows those docs, they might not be able to complete https://docs.seldon.io/projects/seldon-core/en/latest/install/gcp.html#install-seldon-core (seldon pods remain in Pending state, from my experience). giving this heads-up helps those who are not familiar enough with kubernetes to troubleshoot a pending deployment and might conclude, "oh- this doesn't work". i'll let you figure out the best way to convey this if you think it's useful.

**Comment by edshee** [comment]
Out of interest, did you install Istio or Ambassador? I think Istio is the thing that requires 3 nodes (or a big node) as it's pretty resource hungry. I'm pretty sure the Seldon controller would run fine on a single node.

**Comment by vinayvinay** [comment]
Hi Ed, I installed Istio, which explains.

**Comment by edshee** [comment]
Ok great. I merged this earlier so that others wouldn't have the same issue you had. I'll reword it to highlight Istio's node requirement on a subsequent commit.

**Comment by vinayvinay** [comment]
Thanks for updating it to be accurate. I wonder if seldon maintainers have a preference: Istio or Ambassador?
